{
  mockCollaborator.noArgsOnRuntimeExceptionMatch();
  replay(mockCollaborator);
  this.testBean.exceptional(new RuntimeException());
  verify(mockCollaborator);
}
